tests/integration/components/sidenotes-wrapper-test.js

```diff
@@ -0,0 +1,208 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, findAll, render, waitUntil } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { htmlSafe } from '@ember/template';
+
+module('Integration | Component | sidenotes-wrapper', function (hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(async function () {
+    this.set('items', [
+      { id: 1, y: 150, data: null },
+      { id: 2, y: 278, data: null },
+      { id: 3, y: 280, data: null },
+      { id: 4, y: 330, data: null },
+      { id: 5, y: 650, data: null },
+      { id: 6, y: 750, data: null },
+    ]);
+  });
+
+  test('it renders every sidenotes', async function (assert) {
+    await render(hbs`
+      <SidenotesWrapper @items={{this.items}} as |Sidenote item|>
+        <Sidenote @id={{item.id}} @offsetY={{item.y}} @data={{item.data}}>
+          <div data-sidenote></div>
+        </Sidenote>
+      </SidenotesWrapper>
+    `);
+
+    const sidenotes = await findAll('.sidenote [data-sidenote]');
+    assert.equal(sidenotes.length, this.items.length);
```

Comment:
```suggestion
    assert.dom('[data-sidenote]').exists({ count: this.items.length });
```
Use qunit-dom directly

Reply:
https://github.com/concordnow/ember-sidenotes/pull/1/commits/42da655bb8bb112d94cf4ef3606a77c2cbf18e16